integrations/terraform/protoc-gen-terraform-teleport.yaml

```diff
@@ -180,6 +188,13 @@ exclude_fields:
     - "UserSpecV2.CreatedBy"
     - "UserSpecV2.Status"
 
+    # Integration
+    - "IntegrationV1.Status"
+    # only the "github" integration subkind uses credentials currently, but
+    # credentials is excluded intentionally so that we dont have to handle plugin credentials, which are stored separately.
+    - "IntegrationV1.Spec.credentials"
+    - "IntegrationV1.Spec.GitHub"
```

Comment:
why must we ignore the github integration completely? can we add support for this integration, or at least document that it's not supported and track it in an issue? else we will forget about this and will tell to customers: "yes we support integrations in TF" while we're not.